integration-test/tests/cms/about.us.test.ts

```diff
@@ -21,3 +21,20 @@ test('GET /api/cms/v1/about returns correct about us data', async ({ request })
     }
   }
 });
+
+test.describe('unauthorized request with invalid headers', () => {
+  
+  const testData = [
+    { description: 'header is empty', headers: { 'X-API-KEY': '' }},
```

Comment:
Could we create const and reuse in the tests?  because if anything change it will be easier to change in all the tests. and avoid code duplication :)

Reply:
Talking about `X-API-KEY`

Reply:
Yes, good point, I will think about. Also wanted to create consts for urls. I will create a ticket for it.